src/UKHO.ERPFacade.API/UKHO.ERPFacade.API.csproj

```diff
@@ -39,10 +39,9 @@
     <PackageReference Include="Elastic.Apm" Version="1.29.0" />
```

Comment:
Having added Elastic.Apm.NetCoreAll, you can remove the original three Elastic.Apm packages.